pylint/lint/pylinter.py

```diff
@@ -729,8 +729,10 @@ def enable_fail_on_messages(self):
                         self.fail_on_symbols.append(msg.symbol)
 
     def any_fail_on_issues(self):
-        return self.stats is not None and any(
-            x in self.fail_on_symbols for x in self.stats["by_msg"]
+        return (
+            self.stats is not None
+            and self.stats.get("by_msg")
```

Comment:
```suggestion
            and self.stats.get("by_msg") is not None
```
Not doing that create nasty error if `by_msg == ""` or ``by_msg == 0``